### PR TITLE
Add continue application link to details page

### PR DESCRIPTION
**Pull request by cintamani**
From: https://eaflood.atlassian.net/browse/RUBY-723

This moves the action panel in a shared partial and add a link to continue the registration to it.

Screenshot 2019-11-20 at 10 25 52

### CODE REVIEW

File: app/views/renewing_registrations/show.html.erb [hunk 1]
```diff
@@ -281,18 +281,7 @@
...
-          </h2>
-          <ul>
-            <% if @transient_registration.pending_payment? %>
-              <li>
-                <%= link_to t(".actions_box.links.payment"), transient_registration_payments_path(@transient_registration.reg_identifier) %>
-              </li>
-            <% end %>
-          </ul>
-        </div>
+        <%= render "shared/registrations/action_links_panel", resource: @transient_registration%>
```
Comment (cintamani) [inline app/views/renewing_registrations/show.html.erb]:
```suggestion
        <%= render "shared/registrations/action_links_panel", resource: @transient_registration %>
```

File: config/locales/shared/registrations.en.yml [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
+      show:
+        filler: "–"
+        attributes:
+          location:
+            england: "England"
+            wales: "Wales"
+            scotland: "Scotland"
+            northern_ireland: "Northern Ireland"
+            overseas: "Not in the United Kingdom"
+        business_information:
```
Comment (cintamani) [inline config/locales/shared/registrations.en.yml]:
```suggestion
      business_information:
```

File: config/locales/shared/registrations.en.yml [hunk 3]
```diff
@@ -1,21 +1,29 @@
...
+          labels:
```
Comment (cintamani) [inline config/locales/shared/registrations.en.yml]:
```suggestion
        labels:
```

File: config/locales/shared/registrations.en.yml [hunk 4]
```diff
@@ -1,21 +1,29 @@
...
+            location: "Place of business: %{location}"
```
Comment (cintamani) [inline config/locales/shared/registrations.en.yml]:
```suggestion
          location: "Place of business: %{location}"
```

File: app/presenters/base_registration_presenter.rb [hunk 5]
```diff
@@ -21,6 +21,11 @@ def display_convictions_check_message
     end
   end
 
+  def in_progress?
+    # TODO: For now all registrations are submitted in the new system. To update when we build front end flow.
+    false
+  end
+
```
Comment (cintamani) [inline app/presenters/base_registration_presenter.rb]:
```suggestion

```

File: spec/presenters/base_registration_presenter_spec.rb [hunk 6]
```diff
@@ -5,6 +5,12 @@
   let(:view_context) { double(:view_context) }
   subject { described_class.new(registration, view_context) }
 
+  describe "#in_progress?" do
+    it "returns false" do
+      expect(subject).to_not be_in_progress
+    end
+  end
+
```
Comment (cintamani) [inline spec/presenters/base_registration_presenter_spec.rb]:
```suggestion

```

File: app/presenters/base_registration_presenter.rb [hunk 7]
```diff
@@ -21,6 +21,10 @@ def display_convictions_check_message
     end
   end
 
+  def in_progress?
+    # TODO: For now all registrations are submitted in the new system. To update when we build front end flow.
+    false
+  end
```
Comment (Cruikshanks) [inline app/presenters/base_registration_presenter.rb]:
```suggestion
  end
 
```

Looks to be missing a line between the methods

Comment (cintamani) [reply app/presenters/base_registration_presenter.rb]: 🙇‍♀

**Comment by Cruikshanks** [comment]
You got a rubocop issue! 😁 Ping me when fixed and I'll approve.

**Comment by cintamani** [comment]
Today is a painful day 😮

**Pull request merged**